### PR TITLE
feat: localise commands

### PR DESCRIPTION
**Pull request by marzeq**
**In this PR:**
- Code changes were not tested
- Strings were added/removed (new/deleted string keys)

### CODE REVIEW

File: src/lib/imports.ts [hunk 1]
```diff
@@ -60,3 +86,20 @@ export type GetStringFunction = (
...
+
+interface CommandStrings {
+	names: {
+		[key: string]: string
+	}
+	descriptions: {
+		[key: string]: string
+	}
+	options: {
+		[key: string]: {
```
Comment (ImRodry) [inline src/lib/imports.ts:98]:
```suggestion
		[command: string]: {
```

File: src/lib/imports.ts [hunk 2]
```diff
@@ -60,3 +86,20 @@ export type GetStringFunction = (
...
+			[key: string]: {
```
Comment (ImRodry) [inline src/lib/imports.ts:99]:
```suggestion
			[option: string]: {
```

File: src/lib/imports.ts [hunk 3]
```diff
@@ -60,3 +86,20 @@ export type GetStringFunction = (
 	options?: { variables?: Record<string, string | number>; file?: string; lang?: string },
 ) => // eslint-disable-next-line @typescript-eslint/no-explicit-any
 any
+
+interface CommandStrings {
+	names: {
+		[key: string]: string
```
Comment (ImRodry) [inline src/lib/imports.ts:92]:
```suggestion
		[name: string]: string
```

File: src/lib/imports.ts [hunk 4]
```diff
@@ -60,3 +86,20 @@ export type GetStringFunction = (
...
+	}
+	descriptions: {
+		[key: string]: string
```
Comment (ImRodry) [inline src/lib/imports.ts:95]:
```suggestion
		[description: string]: string
```

File: strings/en/commands.json [hunk 5]
```diff
@@ -0,0 +1,329 @@
...
+		"verify": "Automatically checks for changes on your Crowdin profile and gives you the appropriate roles. If your profile is not stored on our database, you can provide it as an argument in the command. To check if your profile is stored, execute `/profile`. Make sure you have DMs enabled from this server so that the bot can send you a message."
+	},
+	"options": {
+		"8ball": {
+			"question": {
+				"name": "question",
+				"description": "The question to ask the bot"
+			}
+		},
+		"feedback": {},
```
Comment (ImRodry) [inline strings/en/commands.json:53]: for the commands with no options you can just leave them out ?

**Review by ImRodry — CHANGES_REQUESTED** [review]

File: src/lib/util.ts [hunk 6]
```diff
@@ -382,9 +384,21 @@ export async function sendHolidayMessage(holidayName: "easter" | "halloween" | "
...
-	return discordLocale.replace("-", "_")
+	discordLocale = discordLocale.replace("-", "_")
+	if (botLocales.includes(discordLocale)) return discordLocale
+	return "en"
+}
+
+export function transformBotLocale(botLocale: string): LocaleString | null {
+	const isLocale = (locale: string): locale is LocaleString => Object.values(Locale).includes(locale as Locale)
+
+	botLocale = botLocale.replace("_", "-")
```
Comment (Syjalo) [inline src/lib/util.ts:399]: `en` won't be converted to `en-US` or `en-GB`

**Review by Syjalo — CHANGES_REQUESTED** [review]

File: src/lib/util.ts (hunk 6 above)
Comment (ImRodry) [reply src/lib/util.ts:399]: That’s intentional, if you don’t pass localizations for a language it uses the default ones

**Comment by Syjalo** [comment]
I think description strings in `strings/en/help.json` can be removed

**Comment by ImRodry** [comment]
They will be after this PR

**Review by ImRodry — APPROVED** [review]
Thank you so much for working on this with me! I'm sure the translators will enjoy this update

**Comment by ImRodry** [comment]
> I think description strings in `strings/en/help.json` can be removed

Decided that we will keep the old system as the help command still allows for longer descriptions which some commands need. Might delete the duplicated strings in the future but it's not urgent, some will stay

**Pull request merged**